src/plots/polar/polar.js

```diff
@@ -532,8 +532,24 @@ proto.updateAngularAxis = function(fullLayout, polarLayout) {
 
     Axes.doTicksSingle(gd, ax, true);
 
+    // angle of polygon vertices in radians (null means circles)
+    // TODO what to do when ax.period > ax._categories ??
```

Comment:
the "angular period > _categories" subplot in your mock looks good - does that mean this TODO is done?

Reply:
That mock _looks_ good, but I'm not it's correct. I still don't what the case `ax.period` > `ax._categories` can and should represent.

Reply:
... and there's an item about this in https://github.com/plotly/plotly.js/issues/2255